### ISSUE TITLE
Develop in develop branch

### ISSUE THREAD

**Issue by matthiaskoenig** [sys-bio/tellurium#135]
There should be a develop branch for development in which feature branches are merged.
The master branch should always be stable and reflect the releases if possible.

**Comment by matthiaskoenig** [comment]
Additional SED-ML & combine functionality will go in the develop branch in the next few days (merged from branch mkoenig).
This will definitely break the build on windows & mac because it requires the latest dev versions of libsedml & libcombine and also has additional requirements, mainly
```
libcombine
```
@kirichoi @0u812 you should make this work on mac and windows than before we can merge it in master and release. But will give you a notice.

**Comment by 0u812** [comment]
I've been working on `libcombine` support in conda. It is available for the Mac in the `sys-bio` channel:
```
conda install -c sys-bio libcombine
```
I am working on the Linux version now. It should be available within the next few days. @matthiaskoenig it would be great if we could make `libcombine` an optional dependency until it becomes more stable and we have a chance to test it (e.g. in some workshops), so if you could wrap the import statements with the usual
```
try:
  import libcombine
except ImportError:
  warnings.warn('libcombine not available. Some features will not work.')
```
that would be great.

**Comment by 0u812** [comment]
Also, since we have paper in submission, we should probably do this "under the radar" until we get the reviews back. By this I mean we shouldn't advertise `libcombine` availability because then the reviewers may test it. We can publish releases with `libcombine` support using the pre-release option. Once we get the reviews back we can of course publish official releases (assuming we don't find any bugs).

**Comment by matthiaskoenig** [comment]
Yes, will use a fallback via the try import. 
The idea is to ship this after the publication. In my opinion no problem with submitted paper. Reviewers can't expect that development is on hold until the paper is accepted.

There are things I am interested in libcombine which we currently not support in tellurium, like the metadata and recording modified/created dates. In my opinion we should be early adopters of libcombine, also to help to make it more stable. The entry resolution from the archives works very well, there are only some smaller issues with metadata
https://github.com/sbmlteam/libCombine/issues/5
But it the current state it already provides the full set of things we currently provide in tellurium.

**Comment by matthiaskoenig** [comment]
Closing this. This is obsolete.

The way to go is to create pull requests from feature branches against master. Kyle & Kiri have to check that the pull requests do not break the notebook and spyder versions (i.e. you both are responsible to make the merges). All tests should run successfully.
